client/my-sites/email/titan-mail-quantity-selection/index.jsx

```diff
@@ -287,151 +221,157 @@ class TitanMailQuantitySelection extends React.Component {
 		);
 	}
 
-	renderPricingDetails() {
-		const { moment, selectedDomain, titanMonthlyProduct, translate } = this.props;
+	renderUnusedMailboxesNotice() {
+		const { maxTitanMailboxCount, selectedDomain, translate } = this.props;
+
+		if ( ! hasTitanMailWithUs( selectedDomain ) ) {
+			return null;
+		}
+
+		const numberOfUnusedMailboxes =
+			maxTitanMailboxCount - getConfiguredTitanMailboxCount( selectedDomain );
+
+		if ( numberOfUnusedMailboxes <= 0 ) {
+			return;
+		}
+
+		const text = translate(
+			'You have %(numberOfMailboxes)d unused mailbox. Do you want to configure it now instead?',
+			'You have %(numberOfMailboxes)d unused mailboxes. Do you want to configure them now instead?',
+			{
+				count: numberOfUnusedMailboxes,
+				args: {
+					numberOfMailboxes: numberOfUnusedMailboxes,
+				},
+				comment: 'This refers to the number of mailboxes purchased that have not been set up yet',
+			}
+		);
+
+		return (
+			<Notice icon="notice" showDismiss={ false } status="is-warning" text={ text }>
+				<NoticeAction
+					external={ ! isEnabled( 'titan/iframe-control-panel' ) }
+					onClick={ this.handleCreateMailbox }
+				>
+					{ translate( 'Finish Setup' ) }
+				</NoticeAction>
+			</Notice>
+		);
+	}
+
+	renderNewMailboxesNotice() {
+		const { moment, selectedDomain, translate } = this.props;
+
+		if ( ! hasTitanMailWithUs( selectedDomain ) ) {
```

Comment:
This condition means that the notice below doesn't show for new purchases (which are most likely to have the standard price), so we still have a gap here.

It may be worth computing `purchaseCost` (or something similar) from `this.props.titanMonthlyProduct.cost_display` when `hasTitanMailWithUs( selectedDomain )` is false.

Reply:
I don't think we should display a notice in that case though. The user will be coming from the `Email Comparison` page where we already show the price for new purchases. This is different from the case where the user decides to add more mailboxes from the `Email` page, as we display no price on that particular page.